smartling-strings-api/src/main/java/com/smartling/api/strings/v2/pto/CreatedStringPTO.java

```diff
@@ -0,0 +1,66 @@
+package com.smartling.api.strings.v2.pto;
+
+
+public class CreatedStringPTO
```

Comment:
You can use [Project Lombok](https://projectlombok.org/) for your DTOs.

Example: https://github.com/Smartling/java-api-sdk/blob/master/smartling-issues-api/src/main/java/com/smartling/api/issues/v2/pto/CountPTO.java